shell/platform/darwin/ios/framework/Source/FlutterAppDelegate.mm

```diff
@@ -8,6 +8,8 @@
 #include "flutter/shell/platform/darwin/ios/framework/Headers/FlutterViewController.h"
 #include "flutter/shell/platform/darwin/ios/framework/Source/FlutterPluginAppLifeCycleDelegate_internal.h"
 
+#pragma GCC diagnostic error "-Woverriding-method-mismatch"
```

Comment:
It seems like this flag should apply to all files. What about setting this in `flutter/shell/platform/darwin/ios/BUILD.gn`?